packages/clay-card/src/CardWithInfo.tsx

```diff
@@ -36,6 +38,11 @@ interface IProps extends React.BaseHTMLAttributes<HTMLDivElement> {
 	 */
 	disabled?: boolean;
 
+	/**
+	 * Determines the style of the card
+	 */
+	displayType?: CardWithInfoDisplayType;
```

Comment:
Just out of curiosity, why did you create a new type instead of adding it directly here? I see no problem with staying that way just to know.

Reply:
I'm just following your pattern in https://github.com/liferay/clay/blob/1a9e591e623274890362b881474e545ffbf07415/packages/clay-card/src/Card.tsx#L18

Reply:
Lol, I probably missed this during the review 🤪, but thanks anyway. Feel free to change something if you think it doesn't make sense.